### PR TITLE
Fix: Card component inline grids

### PR DESCRIPTION
**Pull request by omgaz**
- Fix Safari/Firefox issues with cards
- Remove inline class from card and add to grouping container

cards

### CODE REVIEW

**Comment by OndrejCholeva** [comment]
Can you fix this?
image

File: src/styles/alexandria/Card.scss [hunk 1]
```diff
@@ -18,10 +18,15 @@
...
 
-  &.inline {
-    display: inline-flex;
-    flex-direction: column;
-    margin-right: $card-spacing-etalon;
+  &-grid-container {
+    align-items: stretch;
+    display: flex;
+    flex-flow: row wrap;
+    margin-right: -$card-spacing-etalon;
```
Comment (OndrejCholeva) [inline src/styles/alexandria/Card.scss]: :+1:

**Comment by omgaz** [comment]
> Can you fix this?

Intended. It uses flex boxes align-items to keep the cards a balanced height by row if they don't have an explicit height. Sadly you can't enforce this by row *and* column :P

screen shot 2016-06-29 at 08 39 51
screen shot 2016-06-29 at 08 40 01

**Comment by pphminions** [comment]
LGTM 📇

**Comment by OndrejCholeva** [comment]
Thanks for explanation, LGTM then :credit_card::credit_card:
:credit_card::credit_card:

**Comment by patrick-ivers** [comment]
LGTM

**Pull request merged**